core/src/main/scala/kafka/common/OffsetMetadataAndError.scala

```diff
@@ -34,7 +34,7 @@ object OffsetMetadata {
 
 case class OffsetAndMetadata(offsetMetadata: OffsetMetadata,
                              commitTimestamp: Long = org.apache.kafka.common.requests.OffsetCommitRequest.DEFAULT_TIMESTAMP,
-                             expireTimestamp: Long = org.apache.kafka.common.requests.OffsetCommitRequest.DEFAULT_TIMESTAMP) {
+                             expireTimestamp: Option[Long] = None) {
```

Comment:
We should update the `toString` below to use string interpolation. Currently it is using `%d` incorrectly.